examples/plugin_load_script.py

```diff
@@ -0,0 +1,36 @@
+
+import numpy as np
+import dclab
+import pathlib
+
+# load a single plugin feature
+dclab.load_plugin_feature("/path/to/plugin_example_features.py")
+
+# loading multiple plugins
+for plugin_path in pathlib.Path("my_plugin_directory").rglob("*.py"):
+    dclab.load_plugin_feature(plugin_path)
+
+# load some data
+ds = dclab.new_dataset("path/to/rtdc/file")
+# access the features
+circ_per_area = ds["circ_per_area"]
+circ_times_area = ds["circ_times_area"]
+
+# do some filtering etc.
+ds.config["filtering"]["circ_times_area min"] = 23
+ds.config["filtering"]["circ_times_area max"] = 29
+ds.apply_filter()
+print("Removed {} out of {} events!".format(np.sum(~ds.filter.all), len(ds)))
+
+# save the plugin features in a file
```

Comment:
Instead of exporting to a file (which is already in the documentation), how about creating a nice plot for this example?

Reply:
Check out my lame plot... https://github.com/PinkShnack/dclab/blob/plugin_feature_1/examples/plugin_plot.jpeg
Perhaps there is something nicer that could be shown?

Reply:
It does not really matter as long as it demonstrates plugin features. I would change/remove the title. Please note that for reproducibility, the plot should be generated with this script: https://github.com/ZELLMECHANIK-DRESDEN/dclab/blob/master/examples/generate_example_images.py
Have a look at the other examples to see how I implemented it.